src/OWASPZAPDotNetAPI/OWASPZAPDotNetAPI.Samples/AuthenticatedScanWithFormsAuthentication.cs

```diff
@@ -1,24 +1,4 @@
-﻿/* Zed Attack Proxy (ZAP) and its related class files.

```

Comment:
Nice catch. I should add the license text here as well. Although it is a sample file on how to consume the API, the license should not have been removed. Thanks! I will fix this.